lib/src/components/breadcrumbs/breadcrumbs.dart

```diff
@@ -0,0 +1,324 @@
+  }
+}
+
+///Class for [BreadCrumbsTruncated]
+class BreadCrumbsTruncated extends StatefulWidget {
+  ///Constructor for [BreadCrumbsTruncated]
+  const BreadCrumbsTruncated({
+    super.key,
+    required this.children,
+    required this.rounded,
+  });
+
+  ///Breadcrumb children
+  final List<Widget> children;
+
+  /// {@macro zeta-component-rounded}
+  final bool rounded;
+
+  @override
+  State<BreadCrumbsTruncated> createState() => _BreadCrumbsTruncatedState();
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty<bool>('rounded', rounded));
+  }
+}
+
+class _BreadCrumbsTruncatedState extends State<BreadCrumbsTruncated> {
+  bool _expanded = false;
+
+  @override
+  Widget build(BuildContext context) {
+    final colors = Zeta.of(context).colors;
+
+    return _expanded
+        ? expandedBreadcrumb()
+        : FilledButton(
+            onPressed: () {
+              setState(() {
+                _expanded = true;
+              });
+            },
+            style: ButtonStyle(
+              backgroundColor: MaterialStateProperty.resolveWith((states) {
+                if (states.contains(MaterialState.hovered)) {
+                  return colors.surfaceHovered;
+                }
+                if (states.contains(MaterialState.pressed)) {
+                  return colors.primary.shade10;
+                }
+                if (states.contains(MaterialState.disabled)) {
+                  return colors.surfaceDisabled;
+                }
+                return colors.warm.shade10;
+              }),
+              foregroundColor: MaterialStateProperty.resolveWith((states) {
+                if (states.contains(MaterialState.disabled)) {
+                  return colors.textDisabled;
+                }
+                return colors.textDefault;
+              }),
+              shape: MaterialStatePropertyAll(
+                RoundedRectangleBorder(
+                  borderRadius:
+                      (widget.rounded ? ZetaRadius.minimal : ZetaRadius.none),
+                ),
+              ),
+              side: MaterialStateProperty.resolveWith((states) {
+                if (states.contains(MaterialState.focused)) {
+                  return BorderSide(
+                    width: ZetaSpacing.x0_5,
+                    color: colors.primary.shade100,
+                  );
+                }
+                if (states.isEmpty) {
+                  return BorderSide(color: colors.borderDefault, width: 0.5);
+                }
+                return null;
+              }),
+              padding: MaterialStateProperty.all(EdgeInsets.zero),
+              minimumSize: MaterialStateProperty.all(Size.zero),
+              elevation: const MaterialStatePropertyAll(0),
+            ),
+            child: const Icon(
+              ZetaIcons.more_horizontal_round,
```

Comment:
All these icons should change based on `rounded` prop - so if rounded is true use `more_horizontal_round`, and if false use `more_horizontal_sharp`